opal/mca/btl/openib/btl_openib_component.c

```diff
@@ -2635,23 +2635,10 @@ btl_openib_component_init(int *num_btl_modules,
         goto no_btls;
     }
 
-    /* If we want fork support, try to enable it */
-#ifdef HAVE_IBV_FORK_INIT
-    if (0 != mca_btl_openib_component.want_fork_support) {
-        if (0 != ibv_fork_init()) {
-            /* If the want_fork_support MCA parameter is >0, then the
-               user was specifically asking for fork support and we
-               couldn't provide it.  So print an error and deactivate
-               this BTL. */
-            if (mca_btl_openib_component.want_fork_support > 0) {
-                opal_show_help("help-mpi-btl-openib.txt",
-                               "ibv_fork_init fail", true,
-                               opal_process_info.nodename);
-                goto no_btls;
-            }
-        }
+    /* If fork support is requested, try to enable it */
```

Comment:
i think we can safely remove mca fork param from openib btl and rely on common_verbs mca param to control fork:
* oob/ud was not much in use anyway and we now reviving it
* openib btl fork param did not work and was disabled by default.
* Now that we have many verbs consumers in the same process - maybe we should have a simple way to control it w/o carrying history of non-working flows.

@alinask tried to enable "synonym" as you suggested - but it creates new situations where parameter availability depends on component loading order, i.e. openib loaded after oob which already called to opal_verbs param which leads to mismatch between values of synonim and its original.